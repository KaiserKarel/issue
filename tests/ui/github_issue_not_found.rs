

#[issue::track(url="https://github.com/KaiserKarel/issue/issues/0")]
fn main() {

}