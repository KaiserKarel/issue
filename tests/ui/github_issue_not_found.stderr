error: unable to access https://github.com/KaiserKarel/issue/issues/0
  failed to fetch issue: {"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/issues#get-an-issue"}
 --> $DIR/github_issue_not_found.rs:3:16
  |
3 | #[issue::track(url="https://github.com/KaiserKarel/issue/issues/0")]
  |                ^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/github_issue_not_found.rs:3:1
  |
3 | / #[issue::track(url="https://github.com/KaiserKarel/issue/issues/0")]
4 | | fn main() {
5 | |
6 | | }
  | |_^ consider adding a `main` function to `$DIR/ui/github_issue_not_found.rs`
